src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -248,136 +249,8 @@ def test_views_lti_video_instructor_live_mode_on(
                             ],
                         }
                     },
-                    "type": RAW,
-                },
-                "xmpp": None,
-            },
-        )
-        self.assertEqual(context.get("modelName"), "videos")
-        self.assertEqual(context.get("sentry_dsn"), "https://sentry.dsn")
-        self.assertEqual(context.get("environment"), "test")
-        self.assertEqual(context.get("release"), "1.2.3")
-        self.assertEqual(context.get("player"), "videojs")
-        # Make sure we only go through LTI verification once as it is costly (getting passport +
-        # signature)
-        self.assertEqual(mock_verify.call_count, 1)
-
-    @mock.patch.object(LTI, "verify")
-    @mock.patch.object(LTI, "get_consumer_site")
-    @override_settings(SENTRY_DSN="https://sentry.dsn")
-    @override_settings(RELEASE="1.2.3")
-    @override_settings(VIDEO_PLAYER="videojs")
-    @override_settings(JITSI_ENABLED=True)
-    def test_views_lti_video_instructor_live_mode_on_no_type_specified(
```

Comment:
keep a test to secure how the backend reacts when no type is specified? (although I know the frontend will always specify one)